packages/browser/package.json

```diff
@@ -45,30 +45,8 @@
     "build-api-docs": "npx typedoc --out docs/api/source/api --readme none",
     "build-docs-preview-site": "npm run build-api-docs; cd docs/api; make html"
   },
-  "devDependencies": {
-    "@types/jest": "^26.0.12",
-    "@types/node-fetch": "^2.5.6",
-    "@typescript-eslint/eslint-plugin": "^2.34.0",
-    "@typescript-eslint/parser": "^2.34.0",
-    "concurrently": "^5.3.0",
-    "eslint": "^6.1.0",
-    "eslint-config-prettier": "^6.10.0",
-    "eslint-plugin-prettier": "^3.1.4",
-    "jest": "^25.1.0",
-    "prettier": "2.1.1",
-    "ts-jest": "^25.5.1",
-    "ts-loader": "^8.0.3",
-    "tsc-watch": "^4.2.9",
-    "typedoc": "^0.18.0",
-    "typedoc-plugin-markdown": "^2.4.2",
-    "typescript": "^3.9.7",
-    "webpack": "^4.44.1",
-    "webpack-bundle-analyzer": "^3.8.0",
-    "webpack-cli": "^3.3.12",
-    "webpack-merge": "^4.2.2"
-  },
   "dependencies": {
-    "@inrupt/solid-client-authn-core": "^0.1.4",
+    "@inrupt/solid-client-authn-core": "file:../core",
```

Comment:
Yikes - this doesn't look right. Lerna should be using symbolic links, so I don't think local `file:` references shouldn't ever appear here (my local copy is `"@inrupt/solid-client-authn-core": "^0.1.4",`)...?

Reply:
This is what `lerna link convert` generated, so I assumed that that was how it should be. Looking at [the published package](https://unpkg.com/browse/@inrupt/solid-client-authn-browser@0.1.5-chorededupe-devDependencies-257252491-828.0/package.json) it also seems like it gets set to the correct dependency on publication.

I can't find anything about whether Lerna is supposed to use symbolic links or something like this, so I don't know whether we should revert its own output. Do you know if there's docs anywhere on what this should look like?

Reply:
@pmcb55 I've reverted the `file:` references, and child packages now only have devDependencies with binaries in them. (Except that `-browser` is the only package using Webpack, so all Webpack-related dependencies are in there as well.)

Reply:
Never mind; tests are still failing, and this doesn't actually seem to hoist up that much anymore, so given that I shouldn't spend too much time on this, I'm closing this.